admin/builders/centos64_build.sh

```
@@ -30,88 +32,27 @@ fi
-cd $build_dir/percolator-test
-(set -x; \
-cmake -DTARGET_ARCH=amd64 -DCMAKE_BUILD_TYPE=Release -DGOOGLE_TEST=1 -DCMAKE_INSTALL_PREFIX=./local-usr -DCMAKE_PREFIX_PATH="${build_dir}/${centos_boost}" -DXML_SUPPORT=ON ${src_dir}/percolator; \
-)
-#-----make------
-make -j 4
-
-
-mkdir -p ${build_dir}/converters
-cd ${build_dir}/converters
+echo "cmake percolator ....."
+mkdir -p ${build_dir}/percolator
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)